spark-cassandra-connector/src/main/scala/org/apache/spark/sql/cassandra/ConfCache.scala

```diff
@@ -0,0 +1,133 @@
+package org.apache.spark.sql.cassandra
+
+import com.datastax.spark.connector.cql.CassandraConnectorConf
+import com.datastax.spark.connector.rdd.ReadConf
+import com.datastax.spark.connector.writer.WriteConf
+import org.apache.commons.lang.StringUtils
+
+import scala.collection.mutable
+
+
+object ConfCache {
```

Comment:
I'm not able to run two SparkContexts at the same time, but SQLContext is something else - it is just a wrapper.